added HareerSakhi's profile

HareerSakhi opened the pull request:
Resolve #594

kaiwalyakoparkar reviewed (approved):
LGTM :rocket: Thanks for collaborating and contributing :)

Panquesito7 reviewed (approved):
LGTM. Thank you for your contribution! 👍 🎉

deephunt3r reviewed (approved):
LGTM 👍🏻

HareerSakhi commented:
Hi, I've just updated my image of my profile because I added some stats to my profile README.md. Please send any feedback if I need any. Thank you @deephunt3r @Panquesito7 and @kaiwalyakoparkar for your comments (:

kaiwalyakoparkar reviewed (approved):
Thanks for collaborating and contributing :)

schmelto commented on data.json:
```diff
@@ -1,4 +1,10 @@
 [
+  {
+    "name": "Hareer Sakhi",
+    "githubUsername": "HareerSakhi",
+    "imageUrl": "https://user-images.githubusercontent.com/70807684/126065573-860e71e9-0dce-47fd-bd99-3ad77c9d7c0c.png",
+    "issueNumber": 594
+  },
```
New profiles should be added to the end of the readme, maybe you can open a new pr and adjust this

schmelto commented:
New profiles should be added to the end of the readme, maybe you can open a new pr and adjust this

AvidCoder101 commented:
Hi, @schmelto I have made a new PR with my new username because I have changed my old one. I will put my new one at the end of the file. Thank you (:
